tests/integration/test_opensearch.py

```diff
@@ -491,6 +491,31 @@ def test_route_through_edge(self):
             lambda: not cluster.is_up(), timeout=240
         ), "gave up waiting for cluster to shut down"
 
+    def test_custom_endpoint(self, opensearch_client, opensearch_wait_for_cluster):
+        domain_name = f"opensearch-domain-{short_uid()}"
+        custom_endpoint = "http://localhost:4566/my-custom-endpoint"
```

Comment:
Not sure in which class to put this test, does it belong to the edge routing, or is this basic funtionallity?

Reply:
Yes, imho putting the test here is fine.